File: Data Structures & Algorithms/two-integer-sum/submission-0.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        unordered_map<int,int> m,n;
        int k=0;
        for(auto i: nums)
        {
            if(m[target-i]!=0)
            return {n[target-i],k};
            m[i]++;
            n[i]=k;
            k++;
            
        }
        
    }
};

File: Data Structures & Algorithms/subsets/submission-0.py
class Solution:
    def subsets(self, nums: List[int]) -> List[List[int]]:
        res = []
        curr = []
        def fun(i):

            if i>=len(nums):
                res.append(curr.copy())
                return
            curr.append(nums[i])
            fun(i+1)
            curr.pop()
            fun(i+1)
        fun(0)
        return res

        

        